lib/customer.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'menu2'
+
+class Customer
+
+  attr_reader :menu
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @order_total = []
+  end
+
+  def select_dishes(dish, quantity=1)
+    fail "This dish is not on our menu" unless menu.menu_items.key?(dish)
+    quantity.times do |order| @order << dish
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `order`. You can omit the argument if you don't care about it. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.